htsohm/htsohm.py

```diff
@@ -463,8 +540,13 @@ def worker_run_loop(run_id):
                 material = generate_material(run_id, config['number_of_atom_types'])
             else:
                 print("selecting a parent / running retests on parent / mutating / simulating")
-                parent_id = select_parent(run_id, max_generation=(gen - 1),
-                                                  generation_limit=config['children_per_generation'])
+                
+                if config['selection_mode'] == 'manual':
+                     parent_id = select_parent_interactive(run_id, max_generation=(gen - 1),
+                                                      generation_limit=config['children_per_generation'])
+                else:
+                    parent_id = select_parent(run_id, max_generation=(gen - 1),
+                                                      generation_limit=config['children_per_generation'])
```

Comment:
is there some sort of an error thrown on startup if this is being run as a worker on a node but the config is set to interactive?